htlcswitch/link.go

```diff
@@ -642,9 +648,7 @@ func (l *channelLink) syncChanStates() error {
 			localChanSyncMsg.NextLocalCommitHeight == 1 &&
 			!l.channel.IsPending() {
 
-			log.Infof("ChannelPoint(%v): resending "+
-				"FundingLocked message to peer",
-				l.channel.ChannelPoint())
+			l.log.Infof("resending FundingLocked message to peer")
```

Comment:
do we have a preference as to whether these should capitalized or not? i see that we are doing both atm. maybe just something to keep in mind when adding new ones

Reply:
I converted everything to lower case. I found the format `ChannelLink(1:2:3): line starts lower case` looked best.